src/java/generator/MapGenerator.java

```diff
@@ -333,67 +359,67 @@ private static void interpretArguments(Map<String, String> arguments) {
 			System.exit(0);
 		}
 
-		if (!Arrays.asList("folder-path").stream().allMatch(arguments::containsKey) && !DEBUG) {
+		if (!Stream.of("folder-path").allMatch(arguments::containsKey) && !DEBUG) {
```

Comment:
This is a stream of a single argument. Just call `arguments.containsKey("folder-path")`.

Reply:
Done